Make noarch

BastianZim opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
This makes the package [noarch](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python).

BastianZim committed:
Make noarch

BastianZim committed:
Update build number

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

BastianZim commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.8.6, and conda-forge-pinning 2021.02.09.20.54.49

BastianZim commented on recipe/meta.yaml:
```diff
@@ -10,25 +11,29 @@ source:
   host:
     - pip
-    - python
+    - python >=3.0
```
From https://github.com/bheinzerling/bpemb/blob/7a6f996e0e03072738d694606a6cac2991bcb77d/setup.py#L27 There is no real minimum version that I could find.

BastianZim replied:
Made it python3 only with https://github.com/conda-forge/bpemb-feedstock/pull/2/commits/e5c568b92d48e859e7c20929de8d5f4dc855c691. Let me know if I should add a different minimum version.

BastianZim committed:
Reformat python minimum

According to https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

BastianZim committed:
Merge branch 'make-noarch' of https://github.com/BastianZim/bpemb-feedstock into make-noarch

BastianZim commented:
@conda-forge-admin, please rerender

BastianZim commented:
@conda-forge/bpemb Would be ready for review, thanks!

BastianZim commented:
Just gently bumping this again @ndmaxar @oblute @rluria14
